PI-Adapter-WRAP-Search/src/main/java/tw/fondus/fews/adapter/pi/search/wrap/util/GDALUtils.java

```diff
@@ -0,0 +1,161 @@
+package tw.fondus.fews.adapter.pi.search.wrap.util;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.StringJoiner;
+import java.util.stream.Stream;
+
+import strman.Strman;
+import tw.fondus.commons.util.file.FileType;
+import tw.fondus.commons.util.string.StringUtils;
+
+/**
+ * The tools is base GADL libray to develop.
+ * 
+ * @author shepherd
+ *
+ */
+
+public class GDALUtils {
+
+	private static final String AAIGRID_STRING = "AAIGrid";
+	private static final String GTIFF_STRING = "GTiff";
+
+	/**
+	 * Use in grid files merge.
+	 * 
+	 * @param GDALPath
+	 * @param mergedDataPath
+	 * @param exportPath
+	 * @param fileName
+	 * @param inputFormat
+	 * @throws IOException
+	 */
+	public static void GDALMerge( String GDALPath, String mergedDataPath, String exportPath, String fileName,
```

Comment:
The Class GDALUtils has GDAL already
So name should not use GDAL, example:  executeMerge.